cloud_images/centos7/install_prereqs.sh

```diff
@@ -33,12 +33,23 @@ Type=oneshot
 ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvde /var/lib/mesos
 ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvdf /var/lib/docker
 ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvdg /dcos/volume0
+ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvdh /var/log
```

Comment:
It's not conditional as in it might or might not be mounted. It will always be mounted like this.
The conditional `if [ "$mount_location" = "/var/log" ]; then` just executes some extra steps that migrate the existing log data to the new filesystem.